### DISCUSSION TITLE
Github "<user> doesn’t have any public repositories yet."

### DISCUSSION THREAD

**Discussion by jub0t** [community/community#49425] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

If the value of the `page` query parameter in your profile repository page exceeds the number of pages containing your repositories, a message will be displayed indicating that "[username] doesn’t have any public repositories yet". This message will be shown regardless of the fact that the user may have public repositories.

A more effective approach would be to either redirect the user to the last page that contains the repositories or display an alternate message such as "[username] has fewer public repositories than the specified page number." This would enhance the user experience and provide clearer information about the available repositories.

Example:
![image](https://user-images.githubusercontent.com/57962439/223476969-fd3e5d54-250b-4545-b7b1-8d1cbb516b2c.png)

**Comment by j50h** [comment]
concordo plenamente tambem to tendo bastante dificuldade em relação a isso inclusive sou nova aqui e caí de paraquedas

**Reply by AJAY77751** [reply]
Your GitHub account currently doesn't have any repositories.

**Reply by AJAY77751** [reply]
What are repository

**Reply by AJAY77751** [reply]
Where do I get them from?

**Comment by novaemailbaru01-lab** [comment]
Bug wa

**Comment by novaemailbaru01-lab** [comment]
`github`

**Comment by novaemailbaru01-lab** [comment]
Github
